src/libcrun/kontain_nonkmexec.c

```diff
@@ -0,0 +1,343 @@
+/*
+ * crun - OCI runtime written in C
+ *
+ * Copyright (C) 2020 Kontain Inc.
```

Comment:
Can it be copyrighted to Kontain and still covered by GPL ?  Please check other files, if there is anything like Copyright RedHat then OK

Reply:
All the source files I've looked at have been copyright by the author:
```
/*
 * crun - OCI runtime written in C
 *
 * Copyright (C) 2017, 2018, 2019 Giuseppe Scrivano <giuseppe@scrivano.org>
 * crun is free software; you can redistribute it and/or modify
 * it under the terms of the GNU Lesser General Public License as published by
 * the Free Software Foundation; either version 2.1 of the License, or
 * (at your option) any later version.
```
Does it matter if the copyright is to a real person or a synthetic person?